aidial_analytics_realtime/app.py

```diff
@@ -176,6 +188,10 @@ async def on_log_message(
 
     timestamp = parse_time(request["time"])
 
+    token_usage = message.get("token_usage", None)
+    trace = message["trace"]
```

Comment:
`trace` should be optional here for backwards compatibility?

Reply:
right. fixed